Frogies-Project/Assets/Scripts/Core/BasePrefabsStorage.cs

```diff
@@ -8,8 +9,10 @@ public class BasePrefabsStorage : ScriptableObject
     {
         [SerializeField] private GameObject _sceneItemPrefab;
         [SerializeField] private StatsStorage _statsStorage;
+        [SerializeField] private WaveStorage _waveStorage;
```

Comment:
declared in two places, put it either in Prefab storage or in GlobalSceneManager